docs/develop/update/postgres.rst

```diff
@@ -29,6 +29,8 @@ To enable public access, update the server's Pillar file:
   postgres:
     public_access: True
 
+.. _postgres-add-configuration:
```

Comment:
We use `pg-` instead of `postgres-` for all except `postgres-public-access`, so we can change this and `postgres-public-access` to use `pg-`.